src/main/webapp/about.jsp

```diff
@@ -58,20 +59,40 @@
       <p> <br>
         <strong>Meet the Team!</strong>
         <ul>
+            <table cellpadding = "10" cellspacing = "1">
+              <tr><td align="center"><img src="About-IMG/Default-Profile-IMG.png" width = "150" height = "150"><br>
+                <a href="https://github.com/MarcelRG" target="_blank"><i style="color: #000000" class="fa fa-github" aria-hidden="true"></i></a>
```

Comment:
What is the empty <i> tag for?

WDYT about creating some CSS for these types of links so you can just refer to the class instead of specifying the same color style everywhere?